dist/nix/quick-lint-js.nix

```diff
@@ -15,15 +14,25 @@ mkDerivation {
   pname = "quick-lint-js";
   version = builtins.head (builtins.split "\n" (builtins.readFile ../../version));
 
-  src = ../../.;
+  src = with pkgs; lib.cleanSourceWith {
+    filter = lib.cleanSourceFilter;
+    src = lib.cleanSourceWith {
+      filter = name: type: !(lib.hasSuffix ".nix" name);
+      src = ../../.;
+    };
+  };
   unpackPhase = null;
+  buildInputs = [ pkgs.python310 ];
 
-  nativeBuildInputs = with pkgs; [ cmake ninja ];
+  nativeBuildInputs = with pkgs; [
+    clang_11 #does not work with clang_12 or clang_7
```

Comment:
`clang_11` isn't needed at run-time, so it should be in `buildInputs`, not `nativeBuildInputs`.

Reply:
in order to work clang_11 had to be in ```nativeBuildInputs``` for me